ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-interfaces.j2

```diff
@@ -8,20 +8,20 @@
 
 | Management Interface | description | Type | VRF | IP Address | Gateway |
 | -------------------- | ----------- | ---- | --- | ---------- | ------- |
-{%     for management_interface in management_interfaces | arista.avd.natural_sort %}
-{%         set vrf = management_interfaces[management_interface].vrf | arista.avd.default('default') %}
-| {{ management_interface }} | {{ management_interfaces[management_interface].description }} | {{ management_interfaces[management_interface].type | arista.avd.default('oob') }} | {{ vrf }} | {{ management_interfaces[management_interface].ip_address }} | {{ management_interfaces[management_interface].gateway }} |
+{%     for management_interface in management_interfaces | arista.avd.convert_dicts('name') | arista.avd.natural_sort('name') %}
+{%         set vrf = management_interface.vrf | arista.avd.default('default') %}
+| {{ management_interface.name }} | {{ management_interface.description }} | {{ management_interface.type | arista.avd.default('oob') }} | {{ vrf }} | {{ management_interface.ip_address }} | {{ management_interface.gateway }} |
```

Comment:
we should have variable protection like `if something is arista.avd.defined` like we have in the eos template. In the doc template this can be implemented like `{{ management_interface.description | arista.avd.default('-') }}`, either inline or with a `set` above the printout (like done with `set ipv6` below).
EOS config template already have this.

Reply:
I know this is not part of this refactoring, so I can approve it as is.